crates/starknet-devnet-core/src/starknet/mod.rs

```diff
@@ -1293,7 +1293,7 @@ mod tests {
 
     #[test]
     fn successful_emptying_of_pending_block() {
-        let config = StarknetConfig::default();
+        let config = StarknetConfig { start_time: Some(0), ..Default::default() };
```

Comment:
Does genesis block have timestamp=0? Should it have the actual timestamp or 0?

Reply:
If you explicitly set it to be zero like in this test it will be zero, otherwise it will be actual timestamp

Reply:
Ok, so here we are setting it to be zero just for the purpose of testing, I was under the wrong impression that genesis block timestamp will be zero in production.

> Btw 32bit timestamp limit is 2038 if I'm not wrong

I don't understand; isn't it `2 ** 32 - 1`?

Reply:
we use u64 but it was just fun fact

Reply:
Mystery revealed: the 2038 from a previous message referred to the year 2038. Resolving.